### DISCUSSION TITLE
blog/2021-retrospective

### DISCUSSION THREAD

**Discussion by giscus** [theodorusclarence/theodorusclarence.com#240] [General]
# blog/2021-retrospective

My retrospective of the year of 2021. Achievements, recognitions, and lessons learned.

https://theodorusclarence.com/blog/2021-retrospective

**Comment by theodorusclarence** [comment]
Hello, kindly leave a like (or 5 😄), and comment if you got something to ask! 🚀

**Reply by skidipap** [reply]
Thank you for sharing your story, I hope I can learn from your experience

**Reply by winatungmiharja** [reply]
Such an inspiring story! Motivates me more and more
